spec/check_columns_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative "../lib/check_columns"
+
+describe CheckColumns do
+    it "returns true if column contains a duplicate" do
```

Comment:
```suggestion
    it "returns true if column does not contain a duplicate" do
```

I think you wanted to say